pkgs/development/python-modules/phik/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, pytest-pylint
+, nbconvert
+, jupyter_client
+, numpy
+, scipy
+, pandas
+, matplotlib
+, numba
+}:
+
+buildPythonPackage rec {
+  pname = "phik";
+  version = "0.9.8";
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    sha256 = "c398452c5c1eea153905666b289c6a153712cf3d58811fa41e2bbbd27a65d678";
+  };
+
+  checkInputs = [
+    pytest
+    pytest-pylint
+  ];
+
+  buildInputs = [
```

Comment:
I got a bit muddled in my comment that suggested making these `buildInputs` - these two should be `checkInputs` actually.

Reply:
Should be indeed `buildInputs`.